src/Link.tsx

```diff
@@ -5,11 +5,11 @@ import { useTheme } from './theme';
 
 export interface ILink extends IText<HTMLAnchorElement | HTMLElement> {}
 
-export const Link: React.FunctionComponent<ILink> = props => {
+export const Link = React.forwardRef<HTMLOrSVGElement, ILink>((props, ref) => {
```

Comment:
```suggestion
export const Link = React.forwardRef<HTMLUListElement | HTMLOListElement, ILink>((props, ref) => {
```

Reply:
Um... were you thinking lists?

But also `as` is an exposed prop.

Reply:
Yeah, lists, sorry

Reply:
Here we really might want to allow something other than `<a>` though, right? Or should we remove the 'as' prop here too?

Reply:
You are right in this case, we might want to render a different tag for some reason here.
Let's leave it as is